### PR TITLE
PICARD-1386: Add $title function

### PR DESCRIPTION
**Pull request by virusMac**
# Summary

* This is a…
    * [ ] Bug fix
    * [x] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem
Creating more flexible title-case function, than function in plugin _titlecase_

* JIRA ticket: [PICARD-1386](https://tickets.metabrainz.org/browse/PICARD-1386)
# Solution
Added func_title function

# Action

### CODE REVIEW

File: picard/script.py [hunk 1]
```diff
@@ -846,6 +846,17 @@ def func_ne_any(parser, x, *args):
     """
     return func_not(parser, func_eq_all(parser, x, *args))
 
+def func_title(parser, text):
+    """ Title-case a text """
```
Comment (zas) [inline picard/script.py]: Please make a more complete description of the function, including an example.

Also add a test for it in https://github.com/metabrainz/picard/blob/master/test/test_script.py

**Comment by samj1912** [comment]
Please rebase and squash this before merging.

**Comment by samj1912** [comment]
Also make sure your commit message follows the contribution guidelines

**Comment by zas** [comment]
`git reflog` is your friend

**Comment by samj1912** [comment]
@phw, the commits were still not fixed?

**Comment by phw** [comment]
@samj1912 There was some mess with unrelated commits in the PR in between, but it had been fixed.

**Pull request merged**